test/test_hl_api.py

```diff
@@ -135,6 +135,22 @@ def f1(x):
     assert abs(pop.champion.fitness) == pytest.approx(0.0)
 
 
+def test_finite_evolve(population_params, genome_params, ea_params):
+    def objective(individual):
+        individual.fitness = float(individual.idx)
+        return individual
+
+    pop = cgp.Population(**population_params, genome_params=genome_params)
+    ea = cgp.ea.MuPlusLambda(**ea_params)
+    evolve_params = {
+        "max_generations": np.inf,
+        "min_fitness": -1e-12,
```

Comment:
```suggestion
        "min_fitness": 0.0,
```
the loop exit criterion takes floating point imprecision into account, no need for such funky fitness values ;)